diff --git a/node_modules/@react-native-community/cli-platform-ios/native_modules.rb b/node_modules/@react-native-community/cli-platform-ios/native_modules.rb
index 1b6eece..78445f2 100644
--- a/node_modules/@react-native-community/cli-platform-ios/native_modules.rb
+++ b/node_modules/@react-native-community/cli-platform-ios/native_modules.rb
@@ -43,6 +43,17 @@ def use_native_modules!(config = nil)
   found_pods = []
 
   packages.each do |package_name, package|
+    #PATCH TO DISABLE CONFLICTING MODULES (duplicate symbols)
+    puts ">> package_name #{package_name}"
+  
+    next if %w(
+      react-native-udp 
+      react-native-tcp
+      # ADD ANY MORE CONFLICTING DEPENDENCIES HERE
+      # OR REMOVE THE DEPS ABOVE IF THEY ARE NOT CONFLICTING
+    ).include?(package_name)
+  
+    # PATCH END
     next unless package_config = package["platforms"]["ios"]
 
     podspec_path = package_config["podspecPath"]
